### DISCUSSION TITLE
How to deactivate user account registration?

### DISCUSSION THREAD

**Discussion by Bambus89** [datenstrom/community#276] [Ask a question] [answered]
Hello There, its me again :)

Is there any option to deactivate the User Registration? I am the only User wo logins and other random people have the ability to create accounts but i dont want this.

**Comment by nibreh** [comment, marked as answer]
Hi

Open your file `/system/config/config.ini` and change the line `EditLoginRestrictions: 0` to `EditLoginRestrictions: 1`

Cheers

**Reply by annaesvensson** [reply]
This has been changed in a later version of Datenstrom Yellow, for current settings have a look at datenstrom/community#231.
